### PR TITLE
pdn/pad: remove deprecated versions of these tools

### PR DESCRIPTION
**Pull request by gadfort**
Removes old versions of PDNgen and ICeWall.

### CODE REVIEW

**Comment by maliberty** [comment]
I believe gf12/bp_single is still on the old icewall.  I'll look at converting it so we can move forward with this PR.  I may have some questions :)

**Comment by gadfort** [comment]
@maliberty no rush to merge this in, for PDN it's been more than a year and ICeWall its been about 5-6 months.

**Comment by gadfort** [comment]
@maliberty with the removal from ORFS this is ready to merge

**Comment by maliberty** [comment]
I'll do a secure CI to ensure I didn't miss anything.

File: src/pdn/test/convert.tcl [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# test for config conversion
-source "helpers.tcl"
-
-read_lef Nangate45/Nangate45_tech.lef
-read_lef Nangate45/Nangate45_stdcell.lef
-read_def nangate_gcd/floorplan.def
-
-convert_pdn_config nangate_convert/Nangate45.pdn
```
Comment (maliberty) [inline src/pdn/test/convert.tcl:8]: Do you want to remove convert_pdn_config in another PR?

**Review by maliberty — APPROVED** [review]

**Pull request merged**